README.md

```diff
@@ -34,7 +40,7 @@ export FAROS_DEPLOYMENT_STATUS="<deploy_status>"; \
 export FAROS_DEPLOYMENT_ENV="<environment>"; \
 export FAROS_PIPELINE="<pipeline>"; \
 export FAROS_BUILD="<build>"; \
-curl -s https://raw.githubusercontent.com/faros-ai/faros-events-cli/main/faros_event.sh | bash -s deployment
+./faros_event.sh deployment
```

Comment:
I think we should just delete this example. There are other deployment examples that show this type of usage. (you can also leave it as I can delete it in my PR that updates this README)

Reply:
feel free to change and merge